engine-types/src/types/mod.rs

```diff
@@ -76,22 +76,14 @@ pub struct FtResolveTransferResult {
     pub refund_amount: Balance,
 }
 
-/// Internal errors to propagate up and format in the single place.
-pub enum ErrorKind {
-    ArgumentParseError,
-    InvalidMetaTransactionMethodName,
-    InvalidMetaTransactionFunctionArg,
-    InvalidEcRecoverSignature,
-}
-
 const HEX_ALPHABET: &[u8; 16] = b"0123456789abcdef";
 
 #[allow(dead_code)]
 pub fn bytes_to_hex(v: &[u8]) -> String {
```

Comment:
Should probably just delete this function and replace call to it with `hex::encode`.

Reply:
Kind of unrelated to this PR though. That, and it'll be changed in the future anyways with the change back to serde.